test/__init__.py

```
@@ -19,6 +22,34 @@
+def _ensure_scylla_driver() -> None:
+    """Make scylla-driver importable, installing it on demand.
+
+    scylla-driver isn't baked into the frozen toolchain image, so it's
+    installed here into a cache dir keyed by Python ABI/platform (since
+    that cache dir can outlive a toolchain/interpreter upgrade) and added
+    to sys.path, instead of every test.py run depending on an image
+    rebuild to pick up a new driver version.
+    """
+    try:
+        import cassandra  # noqa: F401
+        return
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)